src/main/java/com/google/sps/model/PlacesClient.java

```diff
@@ -38,18 +35,18 @@
    * @throws PlacesException A custom exception is thrown to signal an error pertaining to the
    *     Places API.
    */
-  List<String> searchNearby(LatLng location, PlaceType placeType, RankBy rankBy)
-      throws PlacesException;
+  String searchNearby(LatLng location, PlaceType placeType, RankBy rankBy) throws PlacesException;
```

Comment:
Update the method documentation.

Also, what if there is more than one result - document that you pick the first one.

Reply:
Done, thanks! I should reread all the documentation and change them to be more accurate.

Reply:
`RESOLVED`